project/ScalazBuild.scala

```diff
@@ -32,7 +32,8 @@ object Scalaz {
     "-Ywarn-unused:_,-imports",
     "-Ywarn-value-discard",
     "-opt:l:inline",
-    "-opt-inline-from:<source>"
+    "-opt-inline-from:<source>",
+    "-Yno-imports"
```

Comment:
if only there were a way this could have been a one-line diff!

Reply:
It was but then you made me scalafmt :D